vm/src/stdlib/math.rs

```diff
@@ -209,6 +209,24 @@ fn math_floor(vm: &VirtualMachine, args: PyFuncArgs) -> PyResult {
     }
 }
 
+fn math_frexp(vm: &VirtualMachine, args: PyFuncArgs) -> PyResult {
```

Comment:
`fn math_frexp(value: PyObjectRef, vm: &VirtualMachine) -> PyResult`?

Reply:
Thanks, I fixed it